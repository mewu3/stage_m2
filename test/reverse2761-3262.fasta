> AY632535|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> HQ234498|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KU720415|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KU955594|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> LC002520|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KU963573|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KF383119|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KF383121|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KF383118|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> KF268948|2761-3262
cagccaccacaacacuagagaggguuacagaacucaagcaaaagggccau
> KF268950|2761-3262
cagccaccacaacacuagagaggguuacagaacucaagcaaaagggccau
> KF268949|2761-3262
cagccaccacaacacuagagaggguuacagaacucaagcaaaagggccau
> KF383115|2761-3262
cagccaccacaacaccagagaggguuacagaacucaggcaaaagggccau
> DQ859059|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> HQ234500|2761-3262
cagccaccacaacaccagagaggguuauagaacucaagugaaagggccau
> KU963574|2761-3262
cagccaccacaacaccagagaggguuauagaacucaagugaaagggccau
> HQ234501|2761-3262
cagccaccacaacaccagagaggguuacagaacucaauugaaagggccau
> KU955591|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> KU955592|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> KU955595|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> KX198134_(modified)|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> KF383116|2761-3262
cagccaccacaacaccagagaggguuacagaacucaagugaaagggccau
> KF383117|2761-3262
cagccaccacaacaccagagaggguuacagaacccaagugaaagggccau
> EU545988|2761-3262
cagccaucacaacaccagagagggcuacaggacucaaaugaaagggccau
> JN860885|2761-3262
cagccaucacaacaccagagagggcuacaggacccaaaugaaagggccau
> KU955593|2761-3262
cagccaucacaacaccagagagggcuacaggacccaaaugaaagggccau
> KJ776791|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU991811|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaauggaagggccau
> KU647676|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU922923|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU922960|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820897|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX087102|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX247646|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156774|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156775|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156776|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX198135|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU321639|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU509998|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX051563|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX197192|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX280026|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU527068|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU729218|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU312312|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365778|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU758877|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU937936|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501215|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365777|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365780|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365779|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU707826|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX087101|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501216|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501217|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX262887|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX247632|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU870645|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU729217|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU926309|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940224|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940228|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940227|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU926310|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU740184|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU761564|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820898|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU955590|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX056898|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820899|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX117076|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU866423|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX185891|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU963796|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU955589|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX253996|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU497555|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU853012|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU853013|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX101060|2761-3262
nnnnnnnnnnnnnnnnnnnnngggcuacaggacccaaaugaaagggccau
> KU744693|2761-3262
cagccaucacaaugccagagagggcuacaggacccaaaugaaagggccau
> KU681081|2761-3262
cagccaucacaacaccagagagggcuacaggacccaaaugaaagggccau
> KX101061|2761-3262
nnnnnnnnnnnnnnnnnnnnngggcuacaggacccaaaugaaagggccau
> KF993678|2761-3262
cagccaucacaacaccagagaggguuacaggacccaaaugaaagggccau
> KU681082|2761-3262
cagccaucacaacaccagagagggcuacaggacucaaaugaaagggccau
> KX101065|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
> KX101064|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101063|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
> HQ234499|2761-3262
cagccaccacaacaccagagagggcuacaggacucaagugaaagggccgu
> KX101066|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101062|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101067|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
> KF383120|2761-3262
cagucaccacaacaccagagaggguuacagaacucaagugaaagggccau
