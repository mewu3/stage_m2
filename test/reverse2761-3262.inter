> KJ776791|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU991811|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaauggaagggccau
> KU647676|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU922923|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU922960|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820897|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX087102|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX247646|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156774|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156775|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX156776|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX198135|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU321639|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU509998|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX051563|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX197192|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX280026|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU527068|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU729218|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU312312|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365778|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU758877|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU937936|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501215|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365777|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365780|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU365779|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU707826|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX087101|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501216|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU501217|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX262887|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX247632|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU870645|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU729217|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU926309|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940224|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940228|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU940227|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU926310|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU740184|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU761564|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820898|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU955590|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX056898|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU820899|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX117076|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU866423|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX185891|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU963796|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU955589|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX253996|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU497555|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU853012|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KU853013|2761-3262
cagccaucacaauaccagagagggcuacaggacccaaaugaaagggccau
> KX101060|2761-3262
nnnnnnnnnnnnnnnnnnnnngggcuacaggacccaaaugaaagggccau
> KU744693|2761-3262
cagccaucacaaugccagagagggcuacaggacccaaaugaaagggccau
> KX101061|2761-3262
nnnnnnnnnnnnnnnnnnnnngggcuacaggacccaaaugaaagggccau
> KX101065|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
> KX101064|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101063|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
> KX101066|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101062|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101067|2761-3262
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnccau
