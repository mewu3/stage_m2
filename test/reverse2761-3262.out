AACACCAGAGAGG	28
AAGCAAAAGGGCC	3
